pdns/recursor_cache.hh

```diff
@@ -30,7 +30,7 @@
 #include <iostream>
 #include "dnsrecords.hh"
 #include <boost/utility.hpp>
-#undef L
+#undef s_log
```

Comment:
I'm pretty sure all the `#undef s_log` can go since `s_log` is unlikely to collide with `boost::multi_index` in the same way `L` did.

Reply:
Ah yes, I caught some of them already, I'll search some some.